jena-arq/src/main/java/org/apache/jena/sparql/util/NodeUtils.java

```diff
@@ -18,12 +18,9 @@
 
 package org.apache.jena.sparql.util;
 
-import java.util.Collection ;
-import java.util.HashSet ;
-import java.util.Iterator ;
-import java.util.Objects;
-import java.util.Set ;
+import java.util.*;
```

Comment:
Isn't it a bit better to keep what we have in import statements, expanding to what's used instead of importing the whole package?